src/main/kotlin/calculator/Operation.kt

```diff
@@ -0,0 +1,18 @@
+package calculator
+
+enum class Operation(
+    val symbol: String,
+    val operate: (Int, Int) -> Int
+) {
+    PLUS("+", { a, b -> a.plus(b) }),
```

Comment:
```suggestion
    PLUS("+", { a, b -> a + b }),
```
연산자 오버로딩 함수를 사용해주셨네요! 위와 같이 나타낼 수도 있어요. 오버로딩된 연산자를 사용할지, 함수를 그대로 사용할지는 그 때 상황에 맞춰 적절히 사용해주시면 될 것 같습니다.

Reply:
코틀린의 기능(연산자 오버로딩 함수)를 사용해보자 라는 생각으로 사용해보았습니다만..
작성하면서 지금 같은 상황에서는 가독성이 떨어지는 것 같다는 생각도 들었습니다.😅